src/librekey/key_store_g10.c

```diff
@@ -779,6 +1121,291 @@ write_seckey(s_exp_t *s_exp, pgp_seckey_t *key)
     return true;
 }
 
+static bool
+write_protected_seckey(s_exp_t *s_exp, pgp_seckey_t *key, const uint8_t *passphrase)
+{
+    char *         type;
+    s_exp_t        raw_s_exp = {};
```

Comment:
I'm thinking we shouldn't introduce more GNU extensions into the code base if we can avoid it.